packages/block-library/src/paragraph/index.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Appending the wp-block-paragraph to before rendering the stored `core/paragraph` block contents.
```

Comment:
```suggestion
 * Append the `wp-block-paragraph` class before rendering the stored `core/paragraph` block contents.
```

Nit: Just a tweak to the wording.